packages/terra-abstract-modal/src/_ModalOverlay.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import classNames from 'classnames/bind';
 import PropTypes from 'prop-types';
-import styles from './ModalOverlay.module.scss';
+import styles from './_ModalOverlay.module.scss';
```

Comment:
Why underscore? In js _ means "private" in scss it means that it's a partial.

https://stackoverflow.com/questions/34889962/why-put-in-front-of-the-file-name-or-in-scss-css

Reply:
Oh really? didn't know that! I'll change back

Reply:
fixed: https://github.com/cerner/terra-framework/pull/1041/commits/b49155fa7bf4ba19ab4ce4b2b0bc5b27c6459ef8